### ISSUE TITLE
wfFindFile  deprecated

### ISSUE THREAD

**Issue by squirrelslair** [dolfinus/3DAlloy#5]
Error msg: 
```
Deprecated
: Use of wfFindFile was deprecated in MediaWiki 1.34. [Called from ThreeDimentionAlloy::parse3DFunc in /home/floten1/wiki.squirrelslair.ca/extensions/3DAlloy/3DAlloy_body.php at line 108] in
/home/floten1/wiki.squirrelslair.ca/includes/debug/MWDebug.php
on line
377
```

There is a second line that uses it. I have a patch worked out that fixes it, hope to submit it shortly.

**Comment by squirrelslair** [comment]
Sent pull request with fix

**Comment by dolfinus** [comment]
Fixed in #8
